.github/workflows/linting.yml

```
@@ -0,0 +1,24 @@
+name: Linting
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+
+    - name: Install dependencies
+      run: |
+        pip install pipenv
+        pipenv install --dev --deploy --system
+    - name: Lint with flake8
+      run: |
+      pylint .
```

I'm stupid, changing it back.
```suggestion
      flake8 .
```